MULTIPLATFORM.md

```diff
@@ -64,6 +64,15 @@ Compose Multiplatform core libraries can be published to local Maven with the fo
 ```bash
 export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version
 ```
+
+*Note:* instead of using an environment variable, it's possible to use new gradle properties:
```

Comment:
Let's remove env. variable way from the Readme (but we should keep the code for backward compatibility)